composer.json

```diff
@@ -31,6 +33,11 @@
     "scripts": {
         "check-cs": "php-cs-fixer fix --dry-run",
         "fix-cs": "php-cs-fixer fix",
-        "test": "phpunit"
+        "sca": "phpstan analyse",
+        "tests": [
```

Comment:
I'm not sure about the name, because these are not tests, but much more. But on the other hand, it's just a name ... :man_shrugging: